src/pull_automerge/core.clj

```diff
@@ -105,6 +103,17 @@
   (println "Get statuses result:" (result :status))
   result)
 
+(defn contains-label [labels label]
+  (> (count (filter #(= (% :name) label) labels)) 0))
+
+(defn filter-by-label [ response-data label ]
+  (filter
+    #(contains-label (% :labels) label)
+    (json/read-str (response-data :body) :key-fn keyword)))
+
+(defn get-oldest-entry [ coll ]
```

Comment:
Since the options in the http request already handle the ordering, we can probably eliminate this like you said.